.tool/lint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+for d in $(find . -type d -not -iwholename '*.git*' -a -not -iname '.tool' -a -not -iwholename '*vendor*'); do
```

Comment:
no reason just following the pattern from cri-o yours works just as well but would need to be updated if we add any code somewhere else...